src/query/encoding.ts

```diff
@@ -21,13 +21,34 @@ export interface EncodingQuery {
   bin?: boolean | BinQuery | ShortEnumSpec;
   scale?: boolean | ScaleQuery | ShortEnumSpec;
 
+  sort?: SortOrder | SortFieldQuery | ShortEnumSpec;
```

Comment:
We don't really have to handle this question right now.  But if you don't want to do that,  don't introduce `SortFieldQuery` yet.  Simply use `SortOrder | SortField`like Vega-Lite because that obviously means we don't care about enumerating it yet.  

Introducing SortFieldQuery without caring about how to structure enumeration seems weird.